src/Book/Book.tsx

```diff
@@ -123,6 +163,9 @@ const Book = (props: Props) => {
 
     return (
         <div className={"book-main-container"}>
+            <ToastContainer position="top-center" autoClose={5000} hideProgressBar={false} newestOnTop={false}
+                            closeOnClick rtl={false} pauseOnFocusLoss draggable pauseOnHover
+            />
```

Comment:
Note that is not necessary to use the `<ToastContainer>` on every screen. You should render `<ToastContainer/> `(and it's configuration) only once on a top-level component, preferable inside the `<div className="App">` on the router. Then you can call the `toast.success` or the `toast.error` within any component/screen inside this main div.